etc/HIProdOfflineConfiguration.py

```diff
@@ -1851,6 +1840,37 @@
                physics_skims=["PbPbEMu", "PbPbZEE", "PbPbZMu", "PbPbHighPtJets", "LogError", "LogErrorMonitor"],
                scenario=hiRawPrimeScenario)
 
+DATASETS = [  "HIPhysicsRawPrime12", "HIPhysicsRawPrime13", "HIPhysicsRawPrime14", "HIPhysicsRawPrime15",
+              "HIPhysicsRawPrime16", "HIPhysicsRawPrime17", "HIPhysicsRawPrime18", "HIPhysicsRawPrime19",
+              "HIPhysicsRawPrime20", "HIPhysicsRawPrime21", "HIPhysicsRawPrime22", "HIPhysicsRawPrime23",
+              "HIPhysicsRawPrime24", "HIPhysicsRawPrime25", "HIPhysicsRawPrime26", "HIPhysicsRawPrime27",
+              "HIPhysicsRawPrime28", "HIPhysicsRawPrime29", "HIPhysicsRawPrime30", "HIPhysicsRawPrime31",
+              "HIPhysicsRawPrime32", "HIPhysicsRawPrime33", "HIPhysicsRawPrime34", "HIPhysicsRawPrime35",
+              "HIPhysicsRawPrime36", "HIPhysicsRawPrime37", "HIPhysicsRawPrime38", "HIPhysicsRawPrime39",
+              "HIPhysicsRawPrime40", "HIPhysicsRawPrime41", "HIPhysicsRawPrime42", "HIPhysicsRawPrime43",
+              "HIPhysicsRawPrime44", "HIPhysicsRawPrime45", "HIPhysicsRawPrime46", "HIPhysicsRawPrime47",
+              "HIPhysicsRawPrime48", "HIPhysicsRawPrime49", "HIPhysicsRawPrime50", "HIPhysicsRawPrime51",
+              "HIPhysicsRawPrime52", "HIPhysicsRawPrime53", "HIPhysicsRawPrime54", "HIPhysicsRawPrime55",
+              "HIPhysicsRawPrime56", "HIPhysicsRawPrime57", "HIPhysicsRawPrime58", "HIPhysicsRawPrime59"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               raw_to_disk=False,
+               aod_to_disk=False,
+               timePerEvent=3,
+               write_nanoaod=False,
+               write_dqm=False,
+               tape_node="T0_CH_CERN_MSS",
+               disk_node="T2_US_Vanderbilt",
+               alca_producers=["EcalUncalZElectron", "EcalUncalWElectron", "MuAlCalIsolatedMu",
```

Comment:
Tbh, instead of lowering the rates of all the producers by running them on less PDs (including the ones that are lightweight and low rate to start with, e.g. the dimuon resonance based once), can't alca turn off or prescale the heavy ones (e.g. `SiStripCalMinBias`) via changing the AlCaReco TriggerBits (a condition in GT)?

Reply:
They considered it but the fastest way to reduce the ALCAs is now to simply remove them from the Tier-0 config.

Reply:
> the fastest way to reduce the ALCAs is now to simply remove them from the Tier-0 config.

How can that be faster than uploading a new payload? That takes 30s

Reply:
If we remove them via the Tier-0 configuration we will not produce them starting from run 388462 (we currently delayed prompt for this reason). If we do it with the trigger bits it will only be effective from newer runs.

We'll deploy it tomorrow, so if there are still concerns, we can discuss it tomorrow morning.

@consuegs FYI

Reply:
>  If we do it with the trigger bits it will only be effective from newer runs.

that's incorrect.
Current FCSR from https://cmsweb.cern.ch/t0wmadatasvc/prod/firstconditionsaferun is 388462.
The alcareco trigger bit tag [AlCaRecoHLTpaths8e29_5e33_v14_prompt](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/tags/AlCaRecoHLTpaths8e29_5e33_v14_prompt) has prompt synchronization, meeting that it will accept a new IOV with since 388462. The two approaches are equivalent in terms of rapidity of deployment, with the DB-based one fulfilling the ultimate goal of having such a mechanism and not penalizing other producer that do not contribute to data volume. Please do consider this.

Reply:
Thanks for the clarification Marco, I wasn't aware it works via the IOV. Let's discuss the details on MM.